tox.ini

```diff
@@ -61,6 +61,7 @@ setenv =
     functests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:5433/lms_functests}
     bddtests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:5433/lms_bddtests}
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
+    VIA_SECRET = not_secret
```

Comment:
I don't know why this bugs me, but we use "notasecret" in `tests/conftest.py` and in most products it's "not_a_secret"